appveyor/compile.ps1

```diff
@@ -185,7 +185,8 @@ call configure.bat ^
 	`"--enable-object-out-dir=$objOutDirectory`" ^
 	`"--with-prefix=$outputDirectory`" ^
 	`"--with-php-build=$dependenciesDirectory`" ^
-	--with-mp=auto
+	--with-mp=auto ^
+	--without-xmlrpc
```

Comment:
It's not a good idea to disable that, there were changes in the last day to bundled libxmlrpc, there may be a problem on windows.